src/commons/GitUtils.ts

```diff
@@ -24,3 +33,13 @@ function getLastMergedFiles(): Set<string> {
   const mergedFiles: string = execSync(LAST_MERGED_FILES_LIST_SCRIPT, { timeout: DEFAULT_TIMOUT }).toString();
   return new Set<string>(mergedFiles.split(/[\r\n]+/));
 }
+
+/**
+ * Return list of all files that mered from hours ago
+ * @param hours - hours to count back
+ */
+function getLastMergedFilesByHours(hours: number): Set<string> {
```

Comment:
this looks like a duplication of getLastMergedFiles function.
you can introduce an optional parameter to getLastMergedFiles and reuse the same function